cmd/hubagent/options/options.go

```diff
@@ -126,6 +128,7 @@ func NewOptions() *Options {
 		EnableV1Alpha1APIs:                      false,
 		EnableClusterInventoryAPIs:              true,
 		EnableStagedUpdateRunAPIs:               true,
+		EnableResourcePlacement:                 true,
```

Comment:
when are we going to remove this? We are adding more and more flags we realistically we won't turn most of them off like the StagedUpdateRun/EnableEvictionAPIs/EnableClusterInventoryAPIs

Reply:
Actually I wonder if we really need to set these here? The `AddFlags` has a default value anyway.